src/i18n/locales/cs-CZ.json

```diff
@@ -6,6 +6,7 @@
   "components.common.languagepicker.chinese": "简体中文",
   "components.common.languagepicker.continueButton": "Choose language",
   "components.common.languagepicker.contributors": "_",
+  "components.common.languagepicker.czech": "Čeština",
```

Comment:
All the `.../locales/xx-XX.json` files, except for `en-US.json`, shouldn't be touched—these are automatically generated by crowdin.